core/emulate_ops.asm

```diff
@@ -210,11 +210,13 @@ fastop_dispatch:
     mov reg_dst, [ebx]
     mov reg_src1, [esi]
     mov reg_src2, [edi]
+    pushf
```

Comment:
I'll suggest move PUSHF to beginning, after push ebp. And move POPF to end, before pop ebp. Then change stack_arg(index) by adding another 0x4 offset. No need to change call stack_arg() so that the fastop handler and remaining arguments are still start from index 0.